### ISSUE TITLE
Regarding the loss_att

### ISSUE THREAD

**Issue by Aboelenien** [ok1zjf/VASNet#20]
In this block in training, I don't get the loss_att and its use.
 
loss_att = 0
loss = criterion(y, target)
loss = loss + loss_att

**Comment by ok1zjf** [comment]
Sorry for the confusion. The loss_att is not being used in this implementation. We experimented with a number
of composite loss functions but didn't use them in the final implementation. You can remove the "loss_att = 0" and 
"loss = loss + loss_att"
